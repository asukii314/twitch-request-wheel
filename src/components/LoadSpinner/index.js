import './load-spinner.css';

function LoadSpinner(props) {
    return <div className="lds-ripple">
        <div></div>
        <div></div>
    </div>;
}
export default LoadSpinner;
