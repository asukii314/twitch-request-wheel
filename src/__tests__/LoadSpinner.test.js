import LoadSpinner from '../components/LoadSpinner';
import {createRenderer} from 'react-test-renderer/shallow';
import React from 'react';

describe('LoadSpinner', () => {

    describe('render', () => {
        test('should render without error', () => {
            const shallowRenderer = createRenderer();
            shallowRenderer.render(<LoadSpinner />);

            let component = shallowRenderer.getRenderOutput();
            expect(component).toMatchSnapshot();
            shallowRenderer.unmount();
        });
    });
});
