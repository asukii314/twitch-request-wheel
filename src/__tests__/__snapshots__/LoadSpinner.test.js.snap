// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`LoadSpinner render should render without error 1`] = `
<div
  className="lds-ripple"
>
  <div />
  <div />
</div>
`;
